sdk/tables/azure-data-tables/inc/azure/data/tables.hpp

```diff
@@ -13,3 +13,4 @@
 #include "azure/data/tables/rtti.hpp"
 #include "azure/data/tables/table_client.hpp"
 #include "azure/data/tables/table_service_client.hpp"
+#include "azure/data/tables/tables_audience.hpp"

```

Comment:
table/tables mix... I am not saying it is incorrect, but maybe we ould move TablesAudience class into `table_client.hpp` - it is anyways useless without including that, and it is one less public API (existence of a header file is also an API) for us to support, plus we don't have to worry about the `table` vs `tables` naming.